utils/helpers.py

```diff
@@ -8,9 +8,7 @@ class YamlDumper(yaml.SafeDumper):
 
     def write_line_break(self, data=None):
         super().write_line_break(data)
-
-        if len(self.indents) == 1:
-            super().write_line_break()
+        super().represent_scalar("tag:yaml.org,2002:str", data, style='"')
```

Comment:
This code was added so that the assert does not fail on quote style.

Prettier will should be applied after this linter to format the file correctly.